keyboards/jj40/keymaps/via/rules.mk

```diff
@@ -0,0 +1,6 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = no
```

Comment:
```suggestion
```
These should work now.

Reply:
Cool! Didn't know about changes in this. Anywhere to read about the available usb interfaces? (is that even the right wording?)